bot/discord/events/message.js

```diff
@@ -1,82 +1,67 @@
-const fetch = require('node-fetch');
-const axios = require('axios');
-const { weirdToNormalChars } = require('weird-to-normal-chars');
+const prefix = config.DiscordBot.Prefix;
+
+const whitelisted = { // commands inside the array means only said commands are whitelisted. if empty then all commands are allowed
+    '898041853096628267': ['info'], // add bot channel
+    '898041866589700128': [], // command channel
+    '898041878447013948': [], // command channel
+    '898041850890440725': [], // command channel
+}
+
+const whitelistedKeys = Object.keys(whitelisted);
+
 module.exports = async (client, message) => {
-    if (message.mentions.users.size >= 20) {
+
+
+    const args = message.content.slice(prefix.length).trim().split(/ +/g);
+
+
+    if (message.mentions.users.size > 0 && [...new Set(message.mentions.users.map(x => x.id))].size >= 20) {
         message.member.ban({ reason: 'Suspected raid. Pinging more than 20 users.' });
         message.channel.send(`${message.member.toString()} has been banned for pinging more than 20 users`);
 
-        const embed = new Discord.MessageEmbed()
+        client.channels.cache.get(config.DiscordBot.oLogs).send(new Discord.MessageEmbed()
             .setTitle("User banned for pinging more than 20 users")
             .addField("User", "Banned " + message.member.toString(), true)
             .setColor(0xFF7700)
-            .setTimestamp(new Date());
-
-        client.channels.cache.get(config.DiscordBot.oLogs).send(embed)
+            .setTimestamp());
     };
+
+
     //Auto reactions on suggestions
-    if (message.channel.id === "898041855135068221" || message.channel.id === "951252958316728340") {
-        if (message.content.includes(">")) {
+    if ((message.channel.id == "898041855135068221" || message.channel.id == "951252958316728340") && !message.content.includes(">")) {
+        await message.react('👍');
+        message.react('👎');
 
-        } else {
-            message.react('👍')
-            setTimeout(() => {
-                message.react('👎')
-            }, 200);
-        }
     }
 
-    if (message.channel.type === "dm") {
-        if (message.author.id === "137624084572798976") {
-            const args = message.content.trim().split(/ +/g);
+    if (message.channel.type == "dm") {
+        if (message.author.id == "137624084572798976")
             client.channels.cache.get(args[0]).send(message.content.split(' ').slice(1).join(' '))
-        } else {
-            if (message.author.id === "640161047671603205") {
+        else if (message.author.id != client.user.id)
+            client.channels.cache.get('898041919022723072').send(message.author.username + " (ID: " + message.author.id +
```

Comment:
Should just use (`) and keep it all in one thing without a ton of (+)'s

Reply:
tho i didnt really change the structure of code, its sometimes a good idea to keep the `+`s since they improve readability if split correctly.